City-of-Pori/pori-kada
# PORI-219 Added domain analytics module and configuration

To test:

1. build the site
2. drush updb -y && drush fra -y && drush cc all
3. Go to /admin/structure/domain/batch/googleanalytics_account and make sure there is set two separate analytics id's for the domains beta.pori.fi and beta.visitpori.fi. Pori version should be UA-76043723-2 and Visitpori version should be UA-76043723-3

The actual functionality of this one can be tested when the code reaches beta servers.
